core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/concat.kt

```diff
@@ -42,6 +42,15 @@ public fun <T, G> GroupBy<T, G>.concat(): DataFrame<G> = groups.concat()
 
 // endregion
 
+// region ReducedGroupBy
+
+public fun <T, G> ReducedGroupBy<T, G>.concat(): DataFrame<G> =
```

Comment:
is this used anywhere? If I look for usages I cannot find any

Reply:
maybe we just need a junit test, we can do it later, just make sure it's in our excel sheet in the right place after moving it here :)

Reply:
i think it was overlooked because it was not in a good place! I will likely revisit it, because sometimes it can produce empty dataframe, need to check that with compiler plugin we'd have a reasonable exception in this case